Frontend_Web_Development_React_Angular_Vue/React/Week 1/Getting_start_with_React.md

```diff
@@ -0,0 +1,149 @@
+<h1 align="center"> ** Getting Started With React** </h1>
+
+## What is React JS?
+
+React JS is one of the most popular and widely used Javascript interfaces to create user interfaces. It is efficient, reusable and flexible and makes the process of coding significantly easier by allowing us to create various components.
+
+## Installation and Setup
+
+To install and setup React JS on your preferred text editor like VSCode,you need to have a recent version of NODE.js installed.Now follow the steps to create a new react app
+
+Step 1: Type the following in the terminal of your text editor to make sure you have Node js successfully installed.
+
+````cd <your-folder-name>
+node```
+
+This command will display a message on your terminal showing the version number of Node installed on your system.
+
+Welcome to Node.js v<your-version-number>.
+Type ".help" for more information.
+
+## Create React App
+
+Step 2: You can now create a new React Application.
+
+In the same terminal,press CTRL+C twice ,type the following command and press enter to create your own blank new react app.
+
+```npx create-react-app my-react-app```
+
+This will create a new React Application.To make changes in the UI,you have to make changes in the ./src folder of your project.
+
+Step 3: To see your application running,type the following command in your terminal and press enter
+
+```cd my-react-app
+npm start```
+
+Step 4: You can now see the demo application running on **http://localhost:3000/** of your default browser.
+
+Now,let's make changes to transform this into a Hello World App.
+
+## Hello World Component with Stateless Functions
+
+Step 1:Create a folder called 'Components' in the ./src folder.
+
+These components can be easily imported in the ./App.js file and thereby rendered on localhost:3000/
+
+Step 2:Inside the ./Components create a new file called 'HelloWorld.jsx'
+
+Step 3:Put the following code in 'HelloWorld.jsx'
+
+````
+
+import React from 'react';
+
+const HelloWorld = () => {
+return (
+
+<div>
+<h1>Hello World</h1>
+</div>
+)
```

Comment:
In the HelloWorld fat arrow function inside that return function **; (semi-colon)** is missing .